op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -115,6 +114,16 @@ var (
 		Usage: "Fail fast on the first error encountered. If set, the db check will stop on the first error encountered, otherwise it will continue to check all blocks and print out all errors at the end.",
 		Value: false,
 	}
+	l1BeaconRPCFlag = &cli.StringFlag{
+		Name:     "l1-beacon-rpc",
+		Usage:    "RPC URL for a node of the L1 beacon chain, required for mainnet migrations but not for alfajores or baklava",
+		Required: false,
+	}
+	l1BeaconchainURLFlag = &cli.StringFlag{
+		Name:     "l1-beaconcha.in-url",
```

Comment:
Is there a fallback service that provides the same API?

Reply:
Not as far as I'm aware, but you can host your own, AFAICS this project is the project serving those pages - https://github.com/gobitfly/eth2-beaconchain-explorer its an indexer/explorer

Reply:
Do you think there could be any reason to communicate with them ahead of the mainnet migration just to make sure they're ready to accommodate all the requests from people migrating and aren't planning any maintenance?

Reply:
It would be good to check for any maintenance issues, in terms of load only two very small requests are being made per migration, so I don't think it would be an issue.